defensics/coap_automation_handler.py

```diff
@@ -71,25 +73,29 @@ def process(self, job):
             # make folder for results
             self.result_path = Path.cwd() / asctime()
             self.result_path.mkdir(parents=True, exist_ok=True)
-            # make newtmgr instance and add connection
-            self.newtmgr = NewtMgr(profile_name='test', conn_type='oic_ble', connstring='"peer_name=c5"')
-            self.newtmgr.make_profile()
+            if crash_detection:
+                # make newtmgr instance and add connection
+                self.newtmgr = NewtMgr(profile_name='test', conn_type='oic_ble', connstring='"peer_name=c5"')
+                self.newtmgr.make_profile()
             logging.debug("Release lock")
             # consume buffered serial data by reading it and saving to /dev/null
-            self.serial_out = SerialOutput(path=devnull,
-                                           newtmgr=self.newtmgr)
-            self.serial_out.start()
-            self.serial_out.shutdown = True
+            if serial_read_enable:
+                self.rtt2pty = RTT2PTY()
+            self.rtt2pty.rtt2pty_start()
```

Comment:
error: "None" has no attribute "rtt2pty_start" :)

Reply:
now it shouldn't